gcc/rust/hir/tree/rust-hir-path.h

```diff
@@ -283,6 +284,7 @@ class PathInExpression : public PathPattern, public PathExpr
   // Constructor
   PathInExpression (Analysis::NodeMapping mappings,
 		    std::vector<PathExprSegment> path_segments,
+		    tl::optional<LangItem::Kind> lang_item = tl::nullopt,
```

Comment:
I'd like to see two constructors, one with segments and one with the lang item. Is that code I wrote or is it something you added for this PR?

Reply:
I added all lang item handling at/below HIR level for PathInExpression.

I can fix this, this was a hack I forgot to remove

Reply:
Fixed